### PR TITLE
[KOGITO-2283] - Shared libaries: PR merging

### PR DESCRIPTION
**Pull request by Kaitou786**
See: https://issues.redhat.com/browse/KOGITO-2283

Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

### CODE REVIEW

File: vars/githubscm.groovy [hunk 1]
```diff
@@ -74,6 +74,18 @@ def mergeSourceIntoTarget(String repository, String sourceAuthor, String sourceB
     """
 }
 
+def mergePR(String credentialID, String PRLink){
```
Comment (Ginxo) [inline vars/githubscm.groovy]: two things here, space after `)`
I would do `def mergePR(String pullRequestLink, String credentialID = 'kie-ci') {`

File: vars/githubscm.groovy [hunk 2]
```diff
@@ -74,6 +74,18 @@ def mergeSourceIntoTarget(String repository, String sourceAuthor, String sourceB
     """
 }
 
+def mergePR(String PRLink, String credentialID='kie-ci') {
```
Comment (Ginxo) [inline vars/githubscm.groovy:77]: `PRLink` should be `pullRequestLink` (or something similar)

Comment (Kaitou786) [reply vars/githubscm.groovy]: changed

File: vars/githubscm.groovy (hunk 1 above)
Comment (Kaitou786) [reply vars/githubscm.groovy]: done

**Comment by Kaitou786** [comment]
@Ginxo done.

**Comment by Ginxo** [comment]
@Kaitou786 few curly brackets are missing, please check

**Comment by Kaitou786** [comment]
@Ginxo sorry about that, please check now ;)

**Pull request merged**